examples/sops/sops_functional_tests_key.asc

```diff
@@ -0,0 +1,174 @@
+-----BEGIN PGP PRIVATE KEY BLOCK-----
```

Comment:
Don't check in the private key in the package as it sets a bad precedence for readers. Maybe add it in a code block in the README with note explaining it's only given for demo purposes.